acstools/utils_calib.py

```diff
@@ -21,6 +21,9 @@
            'from_lt', 'hdr_vals_for_overscan', 'check_oscntab',
            'check_overscan']
 
+# The MJD date of the EVA during SM4 to restore ACS/WFC and ACS/HRC
```

Comment:
Maybe also add a note that this value is also defined in https://github.com/spacetelescope/hstcal/blob/master/pkg/acs/calacs/include/acs.h#L88 , as pointed out by @mdlpstsci .